packages/framework/package.json

```diff
@@ -37,7 +37,7 @@
     "format:fix": "prettier --write --ignore-path .gitignore .",
     "build": "NODE_ENV=production tsup",
     "build:watch": "tsup --watch",
-    "postbuild": "./scripts/copy-package-json.sh && pnpm run check:exports && pnpm check:circulars",
```

Comment:
Copy-package.json is not anymore assuming package exports work correctly. Attw confirms the removal.